components/ldap.rst

```diff
@@ -0,0 +1,66 @@
+.. index::
+   single: Ldap
+   single: Components; Ldap
+
+The Ldap Component
+==================
+
+    The Ldap component provides a means to connect to an LDAP server (OpenLDAP or Active Directory).
```

Comment:
Actually, this is how the word `means` works. In its singular form, it still takes an `s`. :)

Also, as the Ldap component is not the only means to connect to an LDAP server, the article `a` is correct, here. :)